webapps/ui/tasklist/client/scripts/form/directives/cam-tasklist-form-generic-variables.html

```diff
@@ -116,6 +118,26 @@
         <div class="col-xs-4"
              ng-switch="variable.type">
 
+          <div ng-switch-when="Object">
+            <label class="control-label object-type">{{ "CAM_WIDGET_VARIABLE_DIALOG_LABEL_OBJECT_TYPE" | translate }}</label>
+            <input class="form-control"
+                   required
+                   ng-model="variable.valueInfo.objectTypeName"
+                   name="objectTypeName" />
+
+            <label class="control-label">{{ "VARIABLE_OBJECT_TYPE_NAME" | translate }}</label>
```

Comment:
This translate string is not accurate. It says "Object Type Name" while it should say something like "Object Serialization Data Format".
![image](https://user-images.githubusercontent.com/3015690/104184677-1bf43380-5414-11eb-8f8d-58412398c20b.png)